addons/website_forum/tests/test_forum_tag.py

```diff
@@ -0,0 +1,33 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.website_forum.tests.common import TestForumCommon
+
+
+class TestForumTag(TestForumCommon):
+    def setUp(self):
+        super().setUp()
+        self.env['forum.tag'].search([]).unlink()
+        self.tags = self.env['forum.tag'].create(
+            [{'forum_id': self.forum.id, 'name': f'Test Tag {tag_idx}'} for tag_idx in range(1, 3)]
+        )
+        self.env['forum.post'].create(
+            [{'name': 'Posting about tag 1', "forum_id": self.forum.id, "tag_ids": [[6, 0, [self.tags[0].id]]]}]
+        )
+
+    def test_tag_posts_count(self):
+        self._check_tags_post_counts(self.tags, [1, 0])
+        post_tag_1 = self.env['forum.post'].create(
+            [{'name': 'Posting about tag 1 again', "forum_id": self.forum.id, "tag_ids": [[6, 0, [self.tags[0].id]]]}]
+        )
+        self._check_tags_post_counts(self.tags, [2, 0])
+        post_tags = self.env['forum.post'].create(
+            [{'name': 'Posting about tag 2 now', "forum_id": self.forum.id, "tag_ids": [[6, 0, self.tags.ids]]}]
+        )
+        self._check_tags_post_counts(self.tags, [3, 1])
+        post_tag_1.active = False
+        self._check_tags_post_counts(self.tags, [2, 1])
+        post_tags.close(None)
+        self._check_tags_post_counts(self.tags, [1, 0])
+
+    def _check_tags_post_counts(self, tags, expected_counts):
```

Comment:
Would move tools on top of the class (setup, tools, then tests). Just to have the mayonnaise on top, then the frites.